README.md

```diff
@@ -50,6 +50,18 @@ Stay up to date with the latest ONNX news. [[Facebook](https://www.facebook.com/
 
 # Installation
 
+## Prerequisites
+
+```
+numpy >= 1.16.6
+protobuf >= 3.11.3
+six
+typing-extensions>=3.6.2.1
+```
```

Comment:
we should keep this in sync with the ones in setup.py


<!--StartFragment-->install_requires.extend([    'protobuf',    'numpy&gt;=1.16.6',    'six',    'typing&gt;=3.6.4; python_version &lt; "3.5"',    'typing-extensions&gt;=3.6.2.1',])<!--EndFragment-->


in setup.py there is no version mentioned for protobuf we should add it there

Reply:
Good catch! I did it in another PR: https://github.com/onnx/onnx/pull/3448, but I forgot to add it here... Just updated. Thanks.